bepasty/tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+from os import close, unlink
+from random import random
+from tempfile import mkstemp
+
+import pytest
+
+from bepasty.app import create_app, create_storage
+
+
+@pytest.fixture(scope='module')
+def testclient(request):
+    '''
+    creates a Flask-testclient instance for bepasty
+    '''
```

Comment:
please use triple-double quotes for docstrings, some tools have issues with triple-single.